src/tree-selector/treeSelector.html

```diff
@@ -0,0 +1,8 @@
+<miq-tree-view
+  ng-if="$ctrl.data"
+  name="tree-selector"
+  reselect="false"
+  data="{{ $ctrl.data }}"
```

Comment:
just `$ctrl.data`, no `{{` .. we're using `<` since https://github.com/ManageIQ/ui-components/pull/128